src/metrics/cognitive.rs

```diff
@@ -1696,4 +1733,197 @@ mod tests {
             },
         );
     }
+
+    #[test]
+    fn java_no_cognitive() {
+        check_metrics::<JavaParser>("int a = 42;", "foo.java", |metric| {
+            insta::assert_json_snapshot!(
+                metric.cognitive,
+                @r###"
+            {
+              "sum": 0.0,
+              "average": null,
+              "min": 0.0,
+              "max": 0.0
+            }
+            "###
+            );
+        });
+    }
+
+    #[test]
+    fn java_single_branch_function() {
+        check_metrics::<JavaParser>(
+            "class X {
+                public static void print(Boolean a){  
```

Comment:
We should replace `Boolean` with `boolean`

Reply:
Is there a reason for the change by the way? Both are valid. Change made though as I missed this one from previous request. Thanks.

Reply:
Right, forgot to explain the reasons of that choice. `Boolean` is the class that wraps a `boolean` type and it is not automatically imported, so when I had passed the codes to a static analyzer it returned me errors on our test functions. Replacing directly with the type fixed the error and our code remains correct because we use them only for comparisons